craft_providers/bases/buildd.py

```diff
@@ -144,6 +144,8 @@ class BuilddBaseAlias(enum.Enum):
     BIONIC = "18.04"
     FOCAL = "20.04"
     JAMMY = "22.04"
+    KINETIC = "22.10"
+    LUNAR = "23.04"
```

Comment:
There's a devel alias in the buildd-daily I would like to see here
![image](https://user-images.githubusercontent.com/1146853/221040641-06f89b08-4bc7-4093-b660-f9c0e0739ab6.png)

Reply:
This may be out of scope of this PR.  Adding the `DEVEL = "devel"` alias launches the image fine, but raises an error during setup because we are strictly matching `/etc/os-release` against the alias:

```
Incompatible base detected: Expected OS version 'devel', found '23.04'.
```